client/app/home/board/board.js

```diff
@@ -7,10 +8,16 @@ angular.module('artemis.board', [
   $scope.$stateParams = $stateParams;
   $scope.$state = $state;
 
+
   $scope.create = function() {
     $state.go('home.board.create');
   };
 
+  $scope.invite = function() {
+    var boardId = $scope.boardId;
+    $state.go('home.board.invite', {boardId: boardId});
```

Comment:
i think this is unnecessary. You can just do `$state.go('home.board.invite')` and in the `InviteController` you will still have access to the `boardId` from the parent's `$scope`